core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -40,10 +40,32 @@ var (
 	ErrCouldNotGetReceipt = "could not get receipt"
 )
 
+// EvmTxStore combines the txmgr tx store interface and the interface needed for the the API to read from the tx DB
+type EvmTxStore interface {
```

Comment:
Could the `Evm` be dropped?
```suggestion
// TxStore combines the txmgr tx store interface and the interface needed for the the API to read from the tx DB
type TxStore interface {
```

Reply:
might be a little trickier to since there is already a `TxStore` definition and EvmTxStore wraps it. there's a separate ticket to evaluate all the Evm prefixes that I will be getting to later this week :)